orchestrator/forms/validators/customer_contact_list.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019-2023 SURF.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Annotated, Generator, Optional, TypeVar
+from uuid import UUID
+
+from pydantic import BeforeValidator, Field, conlist
+
+from pydantic_forms.validators.components.contact_person import ContactPerson
+from pydantic_forms.validators.helpers import remove_empty_items
+
+T = TypeVar("T")  # pragma: no mutate
+
+
+def customer_contact_list(
+    customer_id: Optional[UUID] = None,
+    customer_key: Optional[str] = "customer",
+    min_items: Optional[int] = None,
+    max_items: Optional[int] = None,
+) -> type[list[T]]:
+    def json_schema_extra() -> Generator:
+        if customer_id:
+            yield "customerId", customer_id
+            yield "organisationId", customer_id
+        if customer_key:
+            yield "customerKey", customer_key
+            yield "organisationKey", customer_key
```

Comment:
Add `# TODO deprecated` comments for organisationId and organisationKey here, and we should have a ticket for i.e. milestone 2.2.0 to actually delete the remaining organisation stuff

Reply:
I think this https://github.com/workfloworchestrator/orchestrator-core/issues/342 ticket is fine for it, since that also still needs to go.

Reply:
That ticket is scoped for milestone 2.1.0 though, that's a bit too soon

Reply:
created a ticket https://github.com/workfloworchestrator/orchestrator-core/issues/484